components/org.wso2.carbon.siddhi.editor.core/src/main/java/org/wso2/carbon/siddhi/editor/core/util/designview/designgenerator/generators/TableConfigGenerator.java

```diff
@@ -55,7 +57,8 @@ public TableConfig generateTableConfig(TableDefinition tableDefinition) throws D
                 tableDefinition.getId(),
```

Comment:
unnecessary new lines. Please format

Reply:
Fixed in 97aff2523a4468759c33ce94e34dd1c88ea092af